codespell_lib/data/dictionary.txt

```diff
@@ -8869,10 +8913,12 @@ noral->normal, moral,
 noralize->normalize
 noralized->normalized
 noraml->normal
+nore->more
 norhern->northern
 normailzation->normalization
 normale->normal
 normales->normals
+normall->normal
```

Comment:
also `normally`

Reply:
I assume you mean as a suggestion @luzpaz ?

Reply:
yes, as a suggestion

Reply:
done